es_base_block.py

```diff
@@ -1,12 +1,15 @@
 from nio.common.block.base import Block
 from nio.common.command.params.dict import DictParameter
+from nio.common.command.params.string import StringParameter
 from nio.metadata.properties.expression import ExpressionProperty
 from nio.metadata.properties.string import StringProperty
 from nio.common.command import command
 from nio.common.discovery import Discoverable, DiscoverableType
 
 
+@command("search", DictParameter("params", default={}))
```

Comment:
Does this work? Putting multiple command calls will add the parameters?

We've typically just listed the parameters inside one command call. See the queue block for an example:
https://github.com/nio-blocks/queue/blob/master/queue_block.py